radix-engine/src/engine/call_frame.rs

```diff
@@ -565,6 +587,30 @@ where
 
         Ok((output, moving_buckets, moving_proofs, moving_vaults))
     }
+
+    pub fn take_cost_unit_counter(&mut self) -> CostUnitCounter {
```

Comment:
Do these need to be `pub`?

Reply:
Looks like it doesn't need to be public. Will remove